### PR TITLE
Added a proposal for my talk

### PR DESCRIPTION
**Pull request by mirhagk**
I started my tenure as an OOP developer, and have been watching functional programming and ADT slowly creep in to everything. I've very much started to see their value compared to OOP approaches, and I'd like to share my thinking with others. Show how to convert OOP ideas to functional programming, and show how to implement modern best practices easier with functional programming.

### CODE REVIEW

**Comment by mirhagk** [comment]
The talk will be very much geared towards OOP programmers. Functional programmers may cringe at my misuse of terms, but I know many OOP developers find the terms very confusing (I know I certainly did)

File: _posts/2016-08-10-CoderCamp-S16E08.md [hunk 1]
```diff
@@ -8,3 +8,4 @@ register: http://codercamp.eventbrite.ca/
 npm install codercampspeakers@16.08.10
 /
```
Comment (rprouse) [inline _posts/2016-08-10-CoderCamp-S16E08.md]: These two lines can be removed now that the first talk is up.

**Comment by rprouse** [comment]
@madmurdock and @bpoetz do you still need a second talk? I've been wanting to talk about .NET Core.

**Comment by bpoetz** [comment]
Yeah, that would be awesome!

**Comment by mirhagk** [comment]
I would enjoy a .NET Core talk as well

**Comment by madmurdock** [comment]
👍

**Comment by bpoetz** [comment]
went ahead and committed + added a note for your talk, @rprouse.  You are committed now!

**Pull request merged**